jujugui/static/gui/src/app/assets/svgs/changes-machine-created.svg

```diff
@@ -1,123 +1,5 @@
-       id="g5687" />
-    <g
-       style="display:inline"
-       id="g5346"
-       transform="translate(-769.06151,404.35792)">
-      <rect
-         y="156.36218"
-         x="952"
-         height="16"
-         width="16"
-         id="rect5348"
-         style="color:#000000;display:inline;overflow:visible;visibility:visible;opacity:1;fill:none;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.5;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:1.5, 1.5;stroke-dashoffset:0.60000002;stroke-opacity:1;marker:none;enable-background:accumulate" />
-      <g
-         style="display:inline"
-         transform="translate(712,-712.00002)"
-         id="g4361">
-        <path
-           sodipodi:nodetypes="ssssssssssssssssss"
-           inkscape:connector-curvature="0"
-           id="path4273"
-           d="m 241.1667,884.3622 c -0.6377,0 -1.1667,-0.52898 -1.1667,-1.16667 l 0,-13.66666 c 0,-0.63769 0.529,-1.16667 1.1667,-1.16667 l 13.6666,0 c 0.6377,0 1.1667,0.52898 1.1667,1.16667 l 0,13.66666 c 0,0.63769 -0.529,1.16667 -1.1667,1.16667 z m 0,-1 13.6666,0 c 0.101,0 0.1667,-0.0657 0.1667,-0.16667 l 0,-13.66666 c 0,-0.10099 -0.066,-0.16667 -0.1667,-0.16667 l -13.6666,0 c -0.101,0 -0.1667,0.0657 -0.1667,0.16667 l 0,13.66666 c 0,0.10099 0.066,0.16667 0.1667,0.16667 z"
-           style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:xx-small;line-height:normal;font-family:sans-serif;-inkscape-font-specification:sans-serif;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;letter-spacing:normal;word-spacing:normal;text-transform:none;direction:ltr;block-progression:tb;writing-mode:lr-tb;baseline-shift:baseline;text-anchor:start;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;color-interpolation:sRGB;color-interpolation-filters:linearRGB;fill:#38b44a;fill-opacity:1;stroke:none;stroke-width:1;marker:none;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" />
-        <rect
-           style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#38b44a;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:2;marker:none;enable-background:accumulate"
-           id="rect4275"
-           width="15.999992"
-           height="1"
-           x="240"
-           y="873.36218" />
-        <rect
-           y="878.36218"
-           x="240"
-           height="1"
-           width="15.999992"
-           id="rect4277"
-           style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#38b44a;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:2;marker:none;enable-background:accumulate" />
-      </g>
-    </g>
-  </g>
+418 bytes - 77.15% saving
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16.000017 16.000017">
```

Comment:
Just so you're aware the build process runs all of the svg's through svg-sprite shrinking them so you don't need to pre-shrink.